statistics/handle/handle_hist.go

```diff
@@ -191,7 +191,7 @@ func (h *Handle) HandleOneTask(lastTask *NeededColumnTask, readerCtx *StatsReade
 	}
 	col := task.TableColumnID
 	oldCache := h.statsCache.Load().(statsCache)
-	tbl, ok := oldCache.Get(col.TableID)
+	tbl, ok := oldCache.GetByQuery(col.TableID)
```

Comment:
I think it's duplicated with GetTableStats.

Reply:
updated.